distribution/build.gradle

```diff
@@ -344,8 +344,12 @@ configure(subprojects.findAll { ['archives', 'packages'].contains(it.name) }) {
           }
         }
         List excludePlatforms = ['linux-x86_64', 'linux-aarch64', 'windows-x86_64', 'darwin-x86_64', 'darwin-aarch64']
-        if (platform != null) {
-          excludePlatforms.remove(excludePlatforms.indexOf(platform))
+        if (os != null) {
+          if (architecture == 'x64') {
+            // ML platform dir uses the x86_64 nomenclature
+            architecture = 'x86_64'
```

Comment:
Not a huge fan of overriding a local argument like this. In this case we don't use it again later in this closure but we _could_ and that would cause funny behavior.

Reply:
Addressed in [1205c8e](https://github.com/elastic/elasticsearch/pull/105842/commits/1205c8efc8002098dda0df302b6f05578fdb39a4)